harvest/utils.py

```diff
@@ -84,6 +86,14 @@ def mark_down(x):
 def has_timezone(date: dt.datetime) -> bool:
     return date.tzinfo is not None and date.tzinfo.utcoffset(date) is not None
 
+def set_system_timezone(date: dt.datetime) -> dt.datetime:
```

Comment:
useful since the user will probably not set the timezone and they usually mean their timezone.